Detectors/EMCAL/reconstruction/include/EMCALReconstruction/Clusterizer.h

```diff
@@ -62,7 +63,8 @@ class Clusterizer
   ~Clusterizer() = default;
 
   void initialize(double timeCut, double timeMin, double timeMax, double gradientCut, bool doEnergyGradientCut, double thresholdSeedE, double thresholdCellE);
-  void findClusters(const std::vector<InputType>& inputArray);
+  //void findClusters(const std::vector<InputType>& inputArray);
```

Comment:
Please remove obsolete code instead of commenting it. Also in a few more places below.